SpatialGDK/Source/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -24,7 +24,23 @@ struct FConnectionConfig
 
 		FString LinkProtocolString;
 		FParse::Value(CommandLine, TEXT("linkProtocol"), LinkProtocolString);
-		LinkProtocol = LinkProtocolString == TEXT("Tcp") ? WORKER_NETWORK_CONNECTION_TYPE_TCP : WORKER_NETWORK_CONNECTION_TYPE_RAKNET;
+		if (LinkProtocolString == TEXT("Tcp"))
+		{
+			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_TCP;
+		}
+		if (LinkProtocolString == TEXT("RakNet"))
+		{
+			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_RAKNET;
+		}
+		if (LinkProtocolString == TEXT("Kcp"))
+		{
+			LinkProtocol = WORKER_NETWORK_CONNECTION_TYPE_KCP;
+		}
+
+#if PLATFORM_PS4 == 1 || PLATFORM_XBOXONE == 1
+		// Use KCP in all cases on console.
```

Comment:
This should probably be changed to essentially disallow RakNet on Console, as RakNet is not compiled for these platforms.